### ISSUE TITLE
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type

### ISSUE THREAD

**Issue by liupzone** [stupidZZ/FCN_Text#3]
Hi stupidZZ:
    I used the caffe version, when I runed python solver.py, it got errors like these:
I0414 16:11:37.903455  8245 layer_factory.hpp:77] Creating layer data
F0414 16:11:37.903491  8245 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ImageLabelmapData (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

how can I solved it?

**Comment by AnameZT** [comment]
@liupzone Hi,I have encountered the same problem recently,I want to ask about that having you solved the problem?Maybe I need some suggestions.
